### PR TITLE
Github action sync to master branch on push

### PR DESCRIPTION
**Pull request by dmzoneill**
Sync devel to master.
the frontend operator pipeline is designed to run on main/master branch push/merge events.
Because patches are landing onto devel, the pipeline won't run, and the latest updates won't appear on stage.
This github action automatically syncs devel to master on merge.

### CODE REVIEW

File: .github/workflows/devel.yml [hunk 1]
```diff
@@ -50,3 +50,25 @@ jobs:
...
+  sync-branches:
+    runs-on: ubuntu-latest
+    name: Syncing branches
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```
Comment (fullsushidev) [inline .github/workflows/devel.yml:63]: why node version 12?

**Review by fullsushidev — CHANGES_REQUESTED** [review]

File: .github/workflows/devel.yml (hunk 1 above)
Comment (fullsushidev) [inline .github/workflows/devel.yml:63]: Shouldn't we use at least v18?

**Review by fullsushidev — DISMISSED** [review]
is there a specific reason why are we not using the latest versions of the actions?

e.g.:
```
codecov/codecov-action@v3
actions/checkout@v4
actions/setup-node@v4
```

File: .github/workflows/devel.yml (hunk 1 above)
Comment (dmzoneill) [reply .github/workflows/devel.yml:63]: This has nothing to do with our code, it has to do with the code in the github action.
The author of the action indicates version 12.  So we use version 12.

**Review by h-kataria — APPROVED** [review]

**Pull request merged**